packages/components/src/components/input/input.scss

```
@@ -15,6 +15,24 @@ $icon-padding: calc(
 .db-input {
 	@include form-components.set-default-form-component(input);
 
+	// increased inline size for elements with icons or specific input types that have icons
+	&:is([data-icon], [data-icon-after]):not([data-hide-icon="true"]),
+	&:has(
+			input[type="search"],
+			input[type="date"],
+			input[type="month"],
+			input[type="week"],
+			input[type="datetime-local"],
+			input[type="time"]
+		) {
+		min-inline-size: variables.$db-sizing-xl;
+	}
+
+	// even larger size when both - leading and trailing icons - are present
+	&[data-icon][data-icon-after] {
+		min-inline-size: form-components.$db-min-inline-size;
```

```suggestion
		--db-form-component-min-inline-size: #{variables.$db-sizing-2xl};
```

We need to use 2xl here right?